### ISSUE TITLE
Login Portion is not working completely

### ISSUE THREAD

**Issue by SubhanRaj** [SubhanRaj/iDiscuss#8]
The login modal in dev branch is as follows

``` PHP

<!-- Modal -->
<div class="modal fade" id="loginModal" tabindex="-1" aria-labelledby="loginModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h5 class="modal-title" id="loginModalLabel">Login to iDiscuss</h5>
                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
            </div>
            <form action="/iDiscuss/partials/_handleLogin.php" method="POST">
                <div class="modal-body">
                    <div class="mb-3">
                        <label for="loginEmail" class="form-label">Email address</label>
                        <input type="email" class="form-control" id="loginEmail" name="loginEmail" aria-describedby="emailHelp">
                    </div>
                    <div class="mb-3">
                        <label for="loginPassword" class="form-label">Password</label>
                        <input type="password" class="form-control" id="loginPassword" name="loginPassword">
                    </div>
                </div>
                <div class="modal-footer">
                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                    <button type="submit" class="btn btn-primary">Login</button>
                </div>
            </form>
        </div>
    </div>
</div>

```

* <b> It sends form data to _loginHandle.php using ```POST``` method</b> where it works as follows
``` PHP

<?php
$showError = "false";


if ($_SERVER["REQUEST_METHOD"] == "POST") {
    include '_dbconnect.php';
    $email = $_POST['loginEmail'];
    $password = $_POST['loginPassword'];
    $sql = "SELECT * FROM `users` WHERE `userEmail` = '$email';";
    $result = mysqli_query($conn, $sql);
    echo $result;
    // Show SQL in case of error

    // if (!$result) {
    //     echo "Error: " . $sql . "<br>" . mysqli_error($conn);
    // }


    $numRows = mysqli_num_rows($result);
    if ($numRows == 1) {
        while ($row = mysqli_fetch_assoc($result)) {
            if (password_verify($password, $row['userPass'])) {

                session_start();
                $_SESSION['loggedin'] = true;
                $_SESSION['username'] = $email;
                echo 'Logged in with email: ' . $email;
                header("Location : /iDiscuss/index.php");
                exit();
            } 
            else {
                $showError = "Password is incorrect.";
                echo $showError;
                // header("Location : /iDiscuss/index.php?error=$showError Password = $password");
                // echo 'Password is incorrect.' . $password;
            }
        }
    }
} else {
    $showError = "Invalid username or password";
    echo $showError;
}
?>

 ```
* But it doesn't work as expected

**Comment by SubhanRaj** [comment]
Now users get successfully redirected to index.php, from pull request #9 but still the wrong password problem exist as described in #10

**Comment by SubhanRaj** [comment]
in Issue #10 this problem is also solved through the PR #11
